src/ui/widgets/widgets/StreamSettingsWidget.cpp

```diff
@@ -393,3 +410,25 @@ void StreamSettingsWidget::on_httpHeadersEditBtn_clicked()
     json["headers"] = rJson;
     stream.httpSettings.headers = HttpObject::fromJson(json).headers;
 }
+
```

Comment:
it seems that you didn't format your codebase with clang-format, please consider doing so.